packages/config/config/devices/0x027a/zen21_3.4.json

```diff
@@ -0,0 +1,345 @@
+// Zooz ZEN21
+// Paddle Switch
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN21",
+	"description": "Paddle Switch",
```

Comment:
Could this be changed to "Z-Wave Plus S2 ON/OFF Switch" to match some of the other Zooz switches like the ZEN26?

Reply:
@bepsoccer

Reply:
Z-Wave Plus ON/OFF Switch for Zen21 and Z-Wave Plus Dimmer Switch for the Zen22 would be most appropriate to follow Zooz's naming.  These are not S2 devices.  I can make that tweak.

Reply:
LGTM, their website and switch paperwork shows the latest zen21 switches I bought are "ETL-listed and Z-Wave Plus certified with S2 security" but it could be a mistake on their part.
https://www.thesmartesthouse.com/products/zooz-z-wave-plus-on-off-light-switch-zen21

Reply:
Ahh you are right they added S2 in 3.0, but they've never put S2 in the name like they do for the 26/27.  I think the description matching their advertised naming is what we should stick with here.